src/App/store/AppState.ts

```diff
@@ -194,6 +194,30 @@ export class Project implements BKProject {
     );
   }
 
+  @computed({ keepAlive: true })
+  get isSelected(): boolean {
+    return this.books.some((book: Book) => {
+      return book.isSelected;
```

Comment:
nit: there is a simpler form for this return, i.e. no `{}` and no `return`. Here and below.